plugins/modules/x509_certificate_revocation_info.py

```diff
@@ -0,0 +1,366 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2016-2017, Yanis Guenane <yanis+ansible@guenane.org>
+# Copyright: (c) 2017, Markus Teufelberger <mteufelberger+ansible@mgit.at>
```

Comment:
Not sure if there's much code from me in there, if anything you should at least add yourself here too. :-)

Reply:
Haha, I think I copied that part from the wrong file :D